lib/slack/renderer/flow/install-github-app.js

```diff
@@ -1,20 +1,30 @@
 const ErrorMessage = require('./error-message');
 
 module.exports = class InstallGitHubApp extends ErrorMessage {
-  constructor(githubAppUrl) {
+  constructor({ installLink, ownerType }) {
     super();
-    this.githubAppUrl = githubAppUrl;
+    this.installLink = installLink;
+    this.ownerType = ownerType;
+  }
+  get guidance() {
+    if (!this.ownerType) {
+      return '\n';
+    }
+    if (this.ownerType === 'User') {
+      return '\n_Note: You will need to ask the owner of the repository to install it for you._';
+    }
+    return '\n_Note: You need to be an organization owner to install the app (or ask one to install it for you)._';
```

Comment:
> …install it for you

Should this link to the [GitHub app installation page](http://github.com/apps/slack)?

Reply:
I'm not sure I follow on this one. Why would we link to the generic page here?

Reply:
It's jut not clear what they're asking the owner to do here.  Are they supposed to ask the owner to install the GitHub app? If so, I think we should give them the link to share. Or are they asking them to run the subscribe command?

Reply:
Ah, that's a very good point! I assumed they would copy the link from the button or from the URL bar, but that's definitely not obvious. We should show the link 👍